ldap/servers/slapd/pblock.c

```diff
@@ -312,28 +318,32 @@ slapi_pblock_clone(Slapi_PBlock *pb)
     if (pb->pb_task != NULL) {
         _pblock_assert_pb_task(new_pb);
         *(new_pb->pb_task) = *(pb->pb_task);
+        memset(pb->pb_task, 0, sizeof(slapi_pblock_task));
```

Comment:
This doesn't make sense. If we are cloning the pblock, we need to clone all it's elements. This is just zeroing them all instead, so pblock clone is now effectively pblock new.